src/main/java/Eliz.java

```diff
@@ -0,0 +1,125 @@
+import java.util.Arrays;
+import java.util.Scanner;
+
+public class Eliz {
+    public static void printTasks(Task[] tasks) {
+        for (int i = 0; i < tasks.length; i++) {
+            int numToPrint = i + 1;
+            System.out.println(numToPrint + ".[" + tasks[i].getTaskType() + "]" + "["+ tasks[i].getStatusIcon() + "] "
+                    + tasks[i].description);
+        }
+    }
+
+    public static void markATask(String line, Task[] tasks) {
+        char taskNumChar = line.charAt(line.length() - 1);
+        int taskNumInt = Character.getNumericValue(taskNumChar);
+        tasks[taskNumInt - 1].setAsDone();
+        System.out.println("[" + tasks[taskNumInt - 1].getTaskType() + "]" + "[X] "
+                + tasks[taskNumInt - 1].description);
+    }
+
+    public static void unmarkATask(String line, Task[] tasks) {
+        char taskNumChar = line.charAt(line.length() - 1);
+        int taskNumInt = Character.getNumericValue(taskNumChar);
+        tasks[taskNumInt - 1].setAsNotDone();
+        System.out.println("[" + tasks[taskNumInt - 1].getTaskType() + "]" + "[ ] "
+                + tasks[taskNumInt - 1].description);
+    }
+
+    public static Task createTask(String line) {
```

Comment:
Can change this to a method in your Task.java class instead? so that your main.java will not be as convoluted